cmd/influx/query.go

```diff
@@ -141,7 +148,6 @@ func fluxQueryF(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
-	results.Release()
```

Comment:
This was already in a `defer` on line 138

Reply:
Its safe and appropriate to call Release multiple times. And it must be called before checking for the error on the next line. 

The defer is there to make sure we release in the case of an early return. This line is there to ensure we have released before we check the error. 

Perhaps a good addition would be to add a comment explaining why both calls to Release exist.